### PR TITLE
Add process cpu and memory usage metrics

### PR DESCRIPTION
**Pull request by mirpo**

### CODE REVIEW

File: src/metrics.js [hunk 1]
```diff
@@ -1,16 +1,23 @@
...
         this._metrics = new Map()
+        this.pidusage = pidusage
     }
 
     createSpeedometer(name) {
         this._metrics.set(name, speedometer())
     }
 
+    async getPidusage(pid = process.pid) {
+        const stats = await this.pidusage(pid)
```
Comment (harbu) [inline src/metrics.js]: Why set `this.pidusage = pidusage` and not just use directly `pidusage(pid)` ?

File: src/logic/Node.js [hunk 2]
```diff
@@ -377,9 +377,10 @@ class Node extends EventEmitter {
         }
     }
 
-    getMetrics() {
+    async getMetrics() {
         const metrics = this.protocols.nodeToNode.endpoint.getMetrics()
 
+        metrics.pidMetrics = await this.metrics.getPidusage()
```
Comment (harbu) [inline src/logic/Node.js]: Maybe move this line inside`Metrics#report()`. That way also tracker will report pid usage.

**Review by harbu — CHANGES_REQUESTED** [review]

File: src/metrics.js (hunk 1 above)
Comment (mirpo) [reply src/metrics.js]: 👍

File: src/logic/Node.js (hunk 2 above)
Comment (mirpo) [reply src/logic/Node.js]: 👍

**Review by harbu — APPROVED** [review]

**Pull request merged**